packages/api/src/utils/logs.js

```diff
@@ -68,10 +68,10 @@ export class Logging {
    * Set user
    *
    * @param {Object} user
-   * @param {number} [user.id]
+   * @param {string} user.id
    */
   setUser(user) {
-    this.metadata.user.id = user.id || 0
+    this.metadata.user.id = user.id
```

Comment:
minor: this removes the default, so we'll start to pass `''` instead of `'0'` where we don't have a user. Both seem odd... I think we should make the call to `this.opts.sentry.setUser` if we have a user id, otherwise skip it.

Reply:
it's probably not an issue in practice as it's all just metadata to sentry, and it has to do a bunch of work to filter out noise from reports anyway. I suspect it would impact our "# users affected" reports for a given bug... with the default to `0` i suspect we would see issues that impact unknown users all logged as impacting just 1.